src/boot_O2/rand.c

```diff
@@ -62,12 +65,19 @@ u32 Rand_Next_Variable(u32* rndNum) {
 /**
  * Generates the next pseudo-random floating-point number between 0.0f and
  * 1.0f from the provided rndNum.
+ *
+ * @remark This is also recommended by Numerical Recipes, pp. 284-5:
+ * > If you need floating-point values instead of 32-bit integers, and want to avoid a divide by floating-point 2^{32},
+ * > a dirty trick is to mask in an exponent that makes the value lie between 1 and 2, then subtract 1.0.
+ * > [...]
+ * > Your authors have tried very hard to make almost all of the material in this book machine and compiler independent
+ * > - indeed, even programming language independent. This subsection is a rare aberration. Forgive us. Once in a great
+ * > while the temptation to be really dirty is just irresistible."
```

Comment:
Probably don't need to quote the book. I think the reference is enough.

Reply:
Oh, I quite liked the quote, I think it adds a bit of flavour and context.

Reply:
fwiw i also think the quote from the book is kinda out of place